### ISSUE TITLE
Low structural tower damping

### ISSUE THREAD

**Issue by profhansen** [IEAWindTask37/IEA-22-280-RWT#37]
Thanks for sharing this new reference turbine(s). I noticed that the structural damping of the first tower modes are set to ~0.5% (logdec). It seems quite low for an offshore turbine. Even without a discrete tower damper, the dissipation in the tower material, connections, and soil-interface it could reach values of about 1-2% (logdec). On top of that comes damping from a tower damper and the water so that the first tower modes of off-shore turbines can have 6% logdec. Maybe some of these sources of damping is modelled by other models?

**Comment by ptrbortolotti** [comment]
hello, thanks for the insights and suggestions. we haven't really got to the point of aligning the dynamic response of tower and monopile as we are still debugging some rotor instabilities in OpenFAST. This said, I'm not 100% sure about the 0.5% log dec value that you report. Can you please elaborate? Is that the tower modes without RNA? In HAWC2 or OpenFAST? 

In HAWC2 I see a comment https://github.com/IEAWindTask37/IEA-22-280-RWT/blob/cb93a64f14126e4d20babaf5a4eeb2cb65df6559/HAWC2/IEA-22-280-RWT-Monopile/htc/iea_22mw_rwt_steps.htc#L45 pointing to 0.5% critical damping (I believe including the RNA @tkbadtu?) 

In OpenFAST, ElastoDyn currently lists 0.5% of critical damping for the tower without RNA, https://github.com/IEAWindTask37/IEA-22-280-RWT/blob/cb93a64f14126e4d20babaf5a4eeb2cb65df6559/OpenFAST/IEA-22-280-RWT-Monopile/IEA-22-280-RWT_ElastoDyn_tower.dat#L5
The IEA15 had 1% for the tower only damping, see here https://github.com/IEAWindTask37/IEA-15-240-RWT/blob/94efa50b53c74e9074977a37f15fe86177a2fc90/OpenFAST/IEA-15-240-RWT-Monopile/IEA-15-240-RWT-Monopile_ElastoDyn_tower.dat#L5, and that's equivalent to 0.3% critical damping including the RNA

**Comment by tkbadtu** [comment]
The damping values in HAWC2 have been tuned for 0.5% damping ratio (~3% log.dec.) for both the tower and monopile as isolated bodies as agreed with NREL. This is considered reasonable without soil interface.

**Comment by profhansen** [comment]
The 0.5% damping ratio is pi % in logdec, and that seems more like a realistic value. I think that there was a DNV rule of thumb of 6% logdec for offshore monopile turbines; so similar to the 1% you IEA15.

@Thanasis, when I computed the standstill modes for the IEA22 model in HAWCStab2, I got 0.09% damping ratio for the first tower modes. I know that the HS2 model is self-made and not in the official repo yet. So maybe we can take it offline.
